cross/curaengine/Makefile

```diff
@@ -0,0 +1,25 @@
+PKG_NAME = curaengine
+PKG_VERS = 15.04
+PKG_EXT = tar.gz
+PKG_DIST_SITE = https://github.com/Ultimaker/CuraEngine/archive
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = CuraEngine-$(PKG_VERS)
+
+DEPENDS = 
+
+HOMEPAGE = https://ultimaker.com
+COMMENT  = CuraEngine is a powerful, fast and robust engine for processing 3D models into 3D printing instruction for Ultimaker and other GCode based 3D printers.
+LICENSE  = AGPLv3
+
+CONFIGURE_TARGET = nope
+INSTALL_TARGET = myInstall
+
+PATCHES_LEVEL = 1
```

Comment:
Is this correct? I recon you do need a Makefile for CuraEngine since it really has to be cross-compiled, or am I mistaken?